From 33669a950aadcff0f0497606c1b475a7930614db Mon Sep 17 00:00:00 2001
From: Pawel Furtak <pawel.furtak@intel.com>
Date: Sun, 25 Nov 2018 17:40:30 +0100
Subject: [PATCH 212/292] ASoC: Intel: Skylake: Register virtualization BE
 service

This patch registers virtualization backend service for
Skylake driver. If virtualization is not enabled,
registration function will do nothing.

Change-Id: Id4e97ef7c53fcd487fca3188162c58adaae399e2
Signed-off-by: Pawel Furtak <pawel.furtak@intel.com>
Reviewed-on:
Reviewed-by: Rojewski, Cezary <cezary.rojewski@intel.com>
Tested-by: Rojewski, Cezary <cezary.rojewski@intel.com>
---
 sound/soc/intel/skylake/skl.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index e87f51d6755e..bd342b14a18f 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -38,6 +38,7 @@
 #include "skl-sst-dsp.h"
 #include "skl-sst-ipc.h"
 #include "skl-topology.h"
+#include "virtio/skl-virtio.h"
 #if IS_ENABLED(CONFIG_SND_SOC_INTEL_SKYLAKE_HDAUDIO_CODEC)
 #include "../../../soc/codecs/hdac_hda.h"
 #endif
@@ -1148,6 +1149,7 @@ static int skl_probe(struct pci_dev *pci,
 		goto out_dsp_free;
 	}
 
+	snd_soc_skl_virtio_miscdev_register(skl);
 	schedule_work(&skl->probe_work);
 
 	return 0;
-- 
2.21.0

